package-scripts/angry-omnibus-toolchain/postrm

```diff
@@ -7,10 +7,12 @@
 #   this programming language.  do not touch.
 # - if you are under 40, get peer review from your elders.
 #
-
-# Clean up the install directory if they exists
-if [ -d /opt/angry-omnibus-toolchain ]; then
-  rm -rf /opt/angry-omnibus-toolchain
+if [ ! -f /etc/redhat-release -a ! -f /etc/fedora-release -a ! -f /etc/system-release -a ! -f /etc/SuSE-release ]; then
```

Comment:
You might be able to simplify this slightly by changing `! -f` to `-d` to check for directory.